udata/core/user/api.py

```diff
@@ -30,6 +34,10 @@
 ns = api.namespace('users', 'User related operations')
 me = api.namespace('me', 'Connected user related operations')
 search_parser = api.search_parser(UserSearch)
+filter_parser = api.parser()
+filter_parser.add_argument(
+    'q', type=unicode, help='The string to filter items',
```

Comment:
Given the `from __future__ import unicode_literals` it should be `str` instead of `unicode`